examples/hackernews/prisma/migrations/20220223114846_comments/migration.sql

```diff
@@ -1,7 +1,6 @@
 -- CreateTable
 CREATE TABLE "Comment" (
     "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
-    "createdAt" DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
What is the motivation behind removing this field? In general, it is always a best practice to have a record of when something was created and introduce pagination in a later advanced chapter.

Reply:
We haven't added it on the tutorial

Reply:
Then, I would propose adding a `createdAt` field to the comment field and sorting the comments returned from the `Link.comments` field by that, so it is used and we don't have to tell people later on: now drop the table and recreate it with a createdAt column or fill the createdAt column with CURRENT_TIMESTAMP.

Reply:
interesting, the goal here is to teach people about the importance of createdAt?
because in that case, creating it without exposing it through GraphQL might better deliver the point, otherwise they might think we are adding it just for sorting?

Reply:
@n1ru4l do you think I should add it on the filtering and pagination chapter or earlier?

Reply:
We can add the createdAt column, field, and order comments by createdAt descending for the Comment when we introduce the comment. Then later on use it further for filtering.

Reply:
Moving our discussion here, don't want to bother you on multiple places:

about the createdAt field, I'm still not sure..
I don't want to add it just because its a best practice without explaining why its a best practice..
I could add that, or I could add that later when we actually need it

I"m not sure if to add it and to write some text about why (which I would love your help with, I can think of some things but maybe there is a good source to explain it better) or to add it in an advanced chapter when we actually need it.

Also, I just found out about this folder:
https://github.com/dotansimha/graphql-yoga/tree/main/website/src/pages/tutorial/advanced
which is not published on the website.

I could work on these and add the createdAt field when the tutorial gets there

Let me know what you think would be best